R/organizations.r

```diff
@@ -8,19 +8,23 @@
 #' 'organization', 'contact', 'endpoint', 'identifier', 'tag', 'machineTag',
 #' 'comment', 'hostedDataset', 'ownedDataset', 'deleted', 'pending',
 #' 'nonPublishing', or the special 'all'. Default: `'all'`
+#' @param country (character) Filters by country as given in isocodes$gbif_name,
+#'  e.g. \code{country="CANADA"}
 #' @param uuid (character) UUID of the data node provider. This must be
-#' specified if data is anything other than 'all'.
+#' specified if data is anything other than 'all', 'deleted', 'pending', or
+#' 'nonPublishing'.
 #' @param query (character) Query nodes. Only used when `data='all'`
 #'
-#' @return A list of length one or two. If `uuid` is NULL, then a
-#' data.frame with call metadata, and a data.frame, but if `uuid` given,
-#' then a list.
+#' @return A list of length of two, consisting of a data.frame \code{meta} when
+#'  uuid is NULL, and \code{data} which can either be a list or a data.frame
+#'  depending on the requested type of data.
 #'
 #' @references <https://www.gbif.org/developer/registry#organizations>
 #'
 #' @examples \dontrun{
 #' organizations(limit=5)
 #' organizations(query="france", limit=5)
+#' organizations(country = "SPAIN")
```

Comment:
This has to be a country code I believe. 
ISO 639-1 (2 letter) country code.

Reply:
Correct, this should also be changed in the r-oxygen template `dataset.r`, it does work most of the time, but not for "SAINT_BARTHELEMY". 

Do you prefer it if users enter isocodes$gbif_name (ANTIGUA_BARBUDA) or isocodes$code (AG)? We can do it either way and still pass the correct argument to the API.

Reply:
I think it's better to use the 2 letter isocode only. This is what is done throughout the rest of rgbif, so I think it should be consistent.

Reply:
Alright, I've changed it in [4ad8ec4](https://github.com/ropensci/rgbif/pull/615/commits/4ad8ec413e255847a9a46efc95f8df48b674897e)